collect_app/src/main/java/org/odk/collect/android/widgets/BearingWidget.java

```diff
@@ -45,26 +50,25 @@
 @SuppressLint("ViewConstructor")
 public class BearingWidget extends QuestionWidget implements BinaryWidget {
     private Button getBearingButton;
-    private TextView answerDisplay;
+    private boolean isSensorAvailable = false;
```

Comment:
It's a false by default you don't need to initialize it.

Reply:
Shall I push another commit?

Reply:
Yes please.